cli/main.go

```diff
@@ -128,6 +157,76 @@ func main() {
 		EnableBashCompletion:   true,
 		UseShortOptionHandling: true,
 		Commands: []*cli.Command{
+			{
+				Name:  "stale-balance",
+				Args:  true,
+				Usage: "If needed, calls `verifyStaleBalances` to correct the balance on a pod you don't own. This will attempt to (1) conclude any existing checkpoint on the pod, and then (2) invoke verifyStaleBalances() to start another, more up-to-date, checkpoint.",
```

Comment:
"calls `verifyStaleBalances` to ~~correct the balance~~ start a checkpoint on a pod you don't own"